app/Enums/PetAdoptionStatus.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Enums;
+
+enum PetAdoptionStatus: string
+{
+    case Available = "available";
+    case Pending = "pending";
+    case Adopted = "adopted";
+    case Quarantined = "quarantined";
+    case Unknown = "unknown";
+
+    public static function values(): array
+    {
+        return array_map(fn($e) => $e->value, self::cases());
```

Comment:
Please check all commited changes and provide expected variable type expected return types
```suggestion
        return array_map(fn(PetAdoptionStatus $e): string => $e->value, self::cases());
```